pkg/kp/flags/kingpin.go

```diff
@@ -19,6 +20,9 @@ func ParseWithConsulOptions() (string, kp.Options) {
 	headers := kingpin.Flag("header", "An HTTP header to add to requests, in KEY=VALUE form. Can be specified multiple times.").StringMap()
 	https := kingpin.Flag("https", "Use HTTPS").Bool()
 	wait := kingpin.Flag("wait", "Maximum duration for Consul watches, before resetting and starting again.").Default("30s").Duration()
+	caFile := kingpin.Flag("tls-ca-file", "File containing the x509 PEM-encoded CA ").ExistingFile()
```

Comment:
The curl options are `--cert`, `--cacert` and `--key`. For at least the last option, `key` is a fairly ambiguous option given that it could refer to a key in the kv store or a gpg signing key. I can remove the `tls-` prefix and I think that will make it sufficiently less verbose.